scrapy/dupefilters.py

```diff
@@ -7,10 +7,6 @@
 
 class BaseDupeFilter:
 
-    @classmethod
-    def from_settings(cls, settings):
-        return cls()
```

Comment:
Come to think of it, this is technically backward incompatible, as users may be calling `super().from_settings()` in their subclasses.